### ISSUE TITLE
DateTimes can only be constructed by parts currently, but there is planned support for parsing. 

### ISSUE THREAD

**Issue by gitfoxi** [quinnj/Datetime.jl#26]
```
julia> Datetime.datetime("yyyy-MM-dd HH:mm:ss", "2011-04-01 00:00:00")
2011-04-01T00:00:00 UTC
```

Looks like the plans are coming right along.

**Comment by quinnj** [comment]
Indeed. I still need to add a little bit more functionality and do some more testing, but the majority of the functionality is up and running, though undocumented.

**Comment by quinnj** [comment]
Fixed by ed8123ede2e316a181a781ecd54fd735657b6cd8
